README.md

```diff
@@ -41,3 +41,12 @@ You can open the demo app that Nginx is proxying by opening a browser to the Ngi
 ```bash
 open "http://$(triton ip nginx_nginx_1)/example"
 ```
+
+### Building
+
+This image implements [microbadger.com](https://microbadger.com/#/labels) label schema, but those labels require additional build args:
+
+```
+docker build --build-arg BUILD_DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"` \
+               --build-arg VCS_REF=`git rev-parse --short HEAD` .
```

Comment:
Is there any way to inject this info into a `docker-compose build`? The command line help doesn't suggest so. Maybe via the compose.yml file? If we break builds using `docker-compose` everywhere that's going to be very annoying for demos.